packages/lodestar/src/db/api/beacon/repository.ts

```diff
@@ -64,12 +64,18 @@ export abstract class BulkRepository<T> extends Repository<T> {
     return (data || []).map((data) => deserialize(data, this.type));
   }
 
-  public async getAllBetween(lowerLimit: number|null, upperLimit: number|null): Promise<T[]> {
+  public async getAllBetween(lowerLimit: number | null, upperLimit: number | null, step?: number | null): Promise<T[]> {
```

Comment:
```suggestion
  public async getAllBetween(lowerLimit: number | null, upperLimit: number | null, step: number | null = null): Promise<T[]> {
```
so you don't have to check for undefineed